src/test_workflow/integ_test/integ_test_suite.py

```diff
@@ -97,6 +97,31 @@ def execute_integtest_sh(self, endpoint: str, port: int, security: bool, test_co
             logging.info(f"{script} does not exist. Skipping integ tests for {self.component.name}")
             return 0
 
+    def multi_execute_integtest_sh(self, endpoint1: str, port1: int, endpoint2: str, port2: int, security: bool, test_config: str) -> int:
```

Comment:
Are we assuming there is only 2 endpoints?

Reply:
Yes, CCR is the only plugin requiring more than one cluster. Since no other plugin uses 

Creating n number of clusters will be an overkill, since we have to refactor a lot of things, the parameters have to be passed dynamically. Since no other plugin/team is using this we are going ahead with just 2 clusters. If you would like, we can rename it to cross_cluster (topology) as we need 2 clusters.

Reply:
We should not assume CCR is the only plugin therefore hardcode the numbers.
Dashboards will soon to use the same functionalities with more than 2 multinode required.

Reply:
Added support to create multiple clusters by specifying the number in topology.